pyqtgraph/Qt/__init__.py

```diff
@@ -258,6 +258,12 @@ def _copy_attrs(src, dst):
 
     if not isinstance(QtOpenGLWidgets, FailedImport):
         QtWidgets.QOpenGLWidget = QtOpenGLWidgets.QOpenGLWidget
+
+    # PySide6 incorrectly placed QFileSystemModel inside QtWidgets
+    if QT_LIB == "PYSIDE6" and hasattr(QtWidgets, 'QFileSystemModel'):
```

Comment:
Remove the quotes from PYSIDE6.
Everywhere else in the library, we (inconsistently) compare directly against 'PySide6', etc. (Case sensitive)

Reply:
🤦🏻  ... yeah a future change should be to convert all the string representations to lower-case (or upper case) before any evaluation.